### PR TITLE
📝 Edit metrics timeframe for Dedicated

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

The sample time for Dedicated metrics is changing to be consistent with Grid. See context.

## What's changed

Updated time frame.

### CODE REVIEW

File: docs/src/increase-observability/metrics/_index.md [hunk 1]
```diff
@@ -71,25 +71,24 @@ Burst allows your container to use additional resources when they aren't require
 
 ## Time intervals
 
-Measurements are taken for each metric _every ten seconds_ for Dedicated environments and _every 1 minute_ for Grid environments.
+Measurements are taken for each metric every 1 minute.
```
Comment (trolologuy) [inline docs/src/increase-observability/metrics/_index.md:74]: :bulb: Do we want to update the video and the first screenshot to reflect that change as well?

File: docs/src/increase-observability/metrics/_index.md [hunk 2]
```diff
@@ -71,25 +71,24 @@ Burst allows your container to use additional resources when they aren't require
...
 You can select a time frame over which to see these measurements for the entire **Metrics** view.
 In the primary three views, averages are shown over larger intervals.
 
 | View                                                                  | Time between measurements                     | Example                      |
 | :-------------------------------------------------------------------- | :-------------------------------------------- | :--------------------------- |
-| The last 15 minutes (*15m*)                                           | 10 seconds for Dedicated, 1 minute for Grid   | 10:00:10, 10:00:20, 10:00:30 |
+| The last 15 minutes (*15m*)                                           | 1 minute                                      | 10:00:10, 10:00:20, 10:00:30 |
```
Comment (trolologuy) [inline docs/src/increase-observability/metrics/_index.md:80]: :lady_beetle:
The example should reflect the timeframe change.
```suggestion
| The last 15 minutes (*15m*)                                           | 1 minute                                      | 10:00, 10:01, 10:02 |
```

**Review by trolologuy — CHANGES_REQUESTED** [review]
Small change to be made before merging.

File: docs/src/increase-observability/metrics/_index.md (hunk 1 above)
Comment (CollierCZ) [reply docs/src/increase-observability/metrics/_index.md:74]: The UI isn't changed yet, so especially the video might be slightly trickier. Not sure it's worth it for something that isn't the focus of the images. :shrug:

**Review by trolologuy — APPROVED** [review]
✅

**Pull request merged**